Utils/git_pull_master_into_fork.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+CONTENT_URL='https://github.com/demisto/content.git'
+CURRENT=$(git branch --show-current)
```

Comment:
let's add an option to provide the branch via an argument, i.e. if the user provided a branch, it'll be checked out, and if he didn't then it'll use the current branch.